etc/stack-schema-map.yaml

```diff
@@ -0,0 +1,12 @@
+# alignment of stack with beats and ecs versions
+#    ECS versions do not align perfectly with stack releases (as of 7.13), so this will reflect MAX ecs version for a
+#    given release
+
+"7.13.0":
+  # beats release about the same time as the stack, so we cannot update this until it is released
+  beats: "7.13.2"
+  ecs: "1.9.0"
+
+"7.14.0":
+  beats: "7.13.2"  # TODO: 7.14.x
```

Comment:
so I'm thinking that we should just have `master` here right?

Reply:
We could just do `if stack_version > max: use beats/ecs master`

This way you won't have to remember as a rule author to bump it. Could even write a rule for multiple stack versions ahead if you wanted. When building the package, we can exempt any rules > current package version

Reply:
1ee8e73